src/PureScript/CST/Lexer.purs

```diff
@@ -36,6 +39,21 @@ import PureScript.CST.Types (Comment(..), IntValue(..), LineFeed(..), ModuleName
 
 infixr 3 alt as <|>
 
+class IsChar a where
+  fromChar :: Char -> a
+  fromCharCode :: Int -> Maybe a
+  singleton :: a -> String
```

Comment:
Is this method used?

Reply:
You're right, I forgot to remove it. Sorry for that